pybamm/models/full_battery_models/equivalent_circuit/thevenin.py

```diff
@@ -30,6 +30,9 @@ class Thevenin(pybamm.BaseModel):
                 throughput capacity in addition to discharge capacity. Must be one of
                 "true" or "false". "false" is the default, since calculating discharge
                 energy can be computationally expensive for simple models like SPM.
+            * "diffusion element" : str
+                Whether to include the diffusion element to the model. Must be one of
+                "true" or "false". "false" is the default.
             * "operating mode" : str
```

Comment:
Can we make this a Boolean rather than a string?

Reply:
We make all options strings so that the types are consistent